Source/drlg_l4.cpp

```diff
@@ -17,7 +17,7 @@ int SP4y1;
 int SP4y2;
 BYTE L4dungeon[80][80];
 BYTE dung[20][20];
-//int dword_52A4DC; // weak
+//int dword_52A4DC;
 

```

Comment:
Why leave this name at all? Was it used in debug build by a chance?

Reply:
We will need it to get memory layout correct.

Reply:
Does this mean we are going to uncomment it when it's location in code is found out?
If so, does this mean this variable was left in the original code unused?

Reply:
That's a lot of questions that I can't really answer at the present time.
If we need to yes. Its location is 52A4DC, hench the name. probably since we don't use it.